app/src/main/java/comcmput301f16t01/github/carrier/MakeRequestActivity.java

```diff
@@ -0,0 +1,212 @@
+package comcmput301f16t01.github.carrier;
+
+import android.app.Activity;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import org.w3c.dom.Text;
+
+import java.util.Locale;
+
+/*
+This will be a long thought...
+
+So I am running into some problems with the Request class so I've made executive
+decisions for now that we may want to change. There is a getEstimate() method in
+the Request class to estimate the method (that calls the getEstimate() method in
+the FareCalculator class. It doesn't make sense to use the Request method here
+because I need the location to create the request (with the constructor) but I
+don't want to make the request partway through the user's activity here because
+they could cancel it. I want to make the Request object when they hit the submit
+button. I don't want to modify the constructor because the start and end locations
+are necessary information for the existence of a Request. Nevertheless, I need to
+show the user the fare estimate on the screen so I need the estimate before I
+instantiate the Request object.
+
+It makes sense for the Request object to require the location parameters because
+those are necessary attributes for a Request to exist. I am assuming that if the
+user doesn't make any changes to the location, the fare estimate will be the same
+every time the method is called so maybe this is a non-issue but it just feels messy
+to have a method that does it, but here I have to call the method directly from its
+own class and then explicitly set the Request fare.
+
+I don't know if this is even a problem or if I am just overthinking things but
+whoever's reviewing my code give me your thoughts or ask me if my use of these
+classes and their methods doesn't make sense...
+ */
```

Comment:
Fair. 
FairEstimate class will no longer be a dependency of Request in this case.